src/appleseedmaya/extensionAttributes.cpp

```diff
@@ -5,6 +5,7 @@
 //
 // This software is released under the MIT license.
 //
+// Copyright (c) 2017 Luis Barrancos, The appleseedhq Organization
```

Comment:
There should be a single copyright notice. See https://github.com/appleseedhq/appleseed/wiki/Copyright-Notices for the detailed explanation.